heron/scheduler-core/tests/java/com/twitter/heron/scheduler/SubmitterMainTest.java

```diff
@@ -124,21 +140,33 @@ public void testSubmitTopology() throws Exception {
     when(config.getStringValue(ConfigKeys.get(UPLOADER_CLASS))).thenReturn(UPLOADER_CLASS);
 
     // Failed to validate the submission
-    doReturn(false).when(submitterMain)
+    doThrow(new TopologySubmissionException(
+        String.format("Topology %s already exist", topology.getName()))).when(submitterMain)
         .validateSubmit(any(SchedulerStateManagerAdaptor.class), anyString());
-    assertFalse(submitterMain.submitTopology());
+
+    try {
+      submitterMain.submitTopology();
+      fail("submitTopology should throw exception");
+    } catch (TopologySubmissionException e) {
+      assertEquals(String.format("Topology %s already exist", topology.getName()), e.getMessage());
```

Comment:
these tests are fixed in #1593. See: [`SubmitterMainTest.java`](https://github.com/twitter/heron/pull/1593/files#diff-9d7c5b93dadaa33d2766bdfdad746d08)